charts/rancher-eks-operator/1.0.801/Chart.yaml

```diff
@@ -4,5 +4,5 @@ description: A Helm chart for provisioning EKS clusters
 home: https://github.com/rancher/eks-operator
 sources:
   - "https://github.com/rancher/eks-operator"
-version: 1.0.800
+version: 1.0.801
```

Comment:
Do we need a questions.yml change in both this chart and the previous chart for rancher min/max version?

Reply:
I don't think so. Rancher auto-upgrades the chart as needed and both chart versions are compatible.

Reply:
The problem is that now the image export script takes any chart that matches the current rancher version, so we do need to cap the old one.